### PR TITLE
Quantile-refactor

### PR DESCRIPTION
**Pull request by dvd101x**
Just a refactor, no new features

### CODE REVIEW

File: src/function/statistics/quantileSeq.js [hunk 1]
```diff
@@ -41,109 +41,112 @@ export const createQuantileSeq = /* #__PURE__ */ factory(name, dependencies, ({
...
+          probArr[i] = _quantileSeq(dataArr, (i + 1) / nPlusOne, sorted)
+        }
+        return probArr
+      }
+    }
 
-          if (probOrN.isNegative()) {
-            throw new Error('N/prob must be non-negative')
-          }
+    if (isBigNumber(probOrN)) {
```
Comment (josdejong) [inline src/function/statistics/quantileSeq.js:80]: I think a nice extra step would be to move the BigNumber implementation in a separate function, like `_quantileSeqProbBigNumber`, what do you think?

Even nicer would be to merge the two implementations into a single, generic one that can handle both `number` and `BigNumber` by replacing operations like `probOrN <= 1` with `smallerEq(probOrN, 1)`, but that is quite some extra work.

**Review by josdejong — COMMENTED** [review]
Nice, way better like this. We can merge this PR as, but I made one inline comment with an idea for further improvement. Can you have a look at that?

**Comment by josdejong** [comment]
I merged #3002, we now have a merge conflict here, can you also resolve that? Thanks!

File: src/function/statistics/quantileSeq.js (hunk 1 above)
Comment (dvd101x) [reply src/function/statistics/quantileSeq.js:80]: No problem, I was wondering the same thing, I will check this and the conflict due to the other merge.

Comment (josdejong) [reply src/function/statistics/quantileSeq.js:91]: @dvd101x do you still want to have a look into extracting the BigNumber logic of quantileSeq in a separate function?

Comment (dvd101x) [reply src/function/statistics/quantileSeq.js:91]: Hi Jos, yes. Unfortunately I haven’t found the solution yet but will have more time to work on it next week. I will report back.

**Comment by dvd101x** [comment]
I just noticed that it's embedded docs are not working since #3002

I will try to fix it in here and then work on #2973 to catch cases like this

**Comment by josdejong** [comment]
Thanks for the updates David! Merging the logic of number and BigNumber worked out really nicely!

**Comment by josdejong** [comment]
The refactor is published now in `v11.11.1`, thanks again David!

**Pull request merged**